Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Stasis Trap perk/scheme

## Changelog:
- Added stasis trap perk which unlocks stasis trap scheme:
    - Stuns character for three years, making them unable to use interactions


## Developer changelog:

## TODO: 
- ~~Event loc. Abduct loc: https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/pull/352#issuecomment-827965376~~ Made by the wonderful KingofMushan


# How to test:
- Try the new scheme:
    - From the actors perspective
    - From the targets perspective (can switch character right before excecuting the scheme as the actor)
    - When it succeeds
    - When it fails
- Make sure no interactions can be used when the character is stunned
- Feel free to tell me if the balance of the scheme is of (too easy/too difficult), or if the stunned modifier needs some sort of balancing

# Test Tips:
- With the `yesman` console command, everyone would like to join your plot.
- Use **Switch Character** from the Main Menu to change character your perspective to another character during the gameplay.
